cryptus-app/pages/testing.tsx

```diff
@@ -1,21 +1,12 @@
 import React from "react";
 import FeatureIamTesting from "../components/template/pagetemplate/pagetemplate";
 import NFTDetails from "../components/wallet_viewer/nft_details/nft_details";
-import { nft } from "../lib/data_types";
+import { nft, profile_props } from "../lib/data_types";
+import get_mock_props from "../lib/get_mock_props";
 
 export default function Home() {
-
-  const mock_nft = {
-    id: 123,
-    collection: "Bored Ape Yacht Club",
-    name: "BAYC #123",
-    description: "BAYC #123",
-    external_url: "https://opensea.io/assets/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d/123",
-    image_url: "https://lh3.googleusercontent.com/ghH08YcdEkukd04RDrn3zAPyFAYpkUSnSoNBZjaeP-LTx6r_v9i6vDaJlbj9a2pbhfPcIi0bOOXIdoFmzrGr9aQcYS6X4XNxFWVJqw=s0",
-    properties: '[{ "name": "Background", "value": "New Punk Blue", "rarity": 0.1232 }, { "name": "Clothes", "value": "Black Holes T", "rarity": 0.0205 }, { "name": "Eyes", "value": "Coins", "rarity": 0.0479 }, { "name": "Hat", "value": "Spinner Hat", "rarity": 0.0181 }, { "name": "Fur", "value": "White", "rarity": 0.0397 }, { "name": "Mouth", "value": "Bored Cigar", "rarity": 0.0121 }]',
-    last_sale_price: 0,
-    last_sale_symbol: "ETH",
-  } as nft
+  const mock_props = get_mock_props() as profile_props;
```

Comment:
what does the as do?

Reply:
Testing.tsx isn't supposed to be modified as I worked on testingMO.tsx